charts.py

```diff
@@ -0,0 +1,145 @@
+import os
+import json
+import numpy as np
+
+from flask import (
+    Flask, flash, render_template,
+    redirect, request, session, url_for)
+from flask_pymongo import PyMongo
+
+if os.path.exists('env.py'):
+    import env
+
+
+charts = Flask(__name__)
+charts.config['MONGO_DBNAME'] = os.environ.get('MONGO_DBNAME')
+charts.config['MONGO_URI'] = os.environ.get('MONGO_URI')
+charts.secret_key = os.environ.get('SECRET_KEY')
+
+mongo = PyMongo(charts)
+
+# collection
+data_set_coll = mongo.db.SampleData
+
+
+@charts.route('/chart_label_one', methods=['GET'])
+def chart_lable_one():
+    # data
+    data_set = list(data_set_coll.find())
+
+    #query search
+    query_activity = "nace_1"
+
+    activity_list = []
+    for data in data_set:
+        activity_list.append(data[query_activity])
+
+    # x-coordinates corresponding to the activities listed with nace code.
+    # here are the code's name corresponding to 'A' and 'J'
+    x_axis = ['AGRICULTURE, FORESTRY AND FISHING',
+              'INFORMATION AND COMMUNICATION']
+
+    # heights of bars
+    # create list of activities list (goes up to 'U' on official website),
+    # but we will do 'A' and 'J' for the moment.
+    a = []
+    j = []
+
+    list_of_act_list = [a,j]
```

Comment:
I would suggest to be more explicit and spell-out activity not to confuse other developer reading your code.

Reply:
Also below.